### PR TITLE
Migration: Add missing job instance in migration

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description (for Contributor and Core Developer)**

Add missing migrations.
- Job instance
- Reindex products & models migration

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Ok
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: upgrades/schema/Version_2_2_20180321103340_index_products_and_product_models.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        $application->setAutoExit(false);
+        $application->run(new ArrayInput([
+            'command' => 'akeneo:elasticsearch:reset-indexes',
+            '--all' => true,
+        ]));
+        $application->run(new ArrayInput([
+            'command' => 'pim:product-model:index',
+            '--all' => true,
+        ]));
+        $application->run(new ArrayInput([
```
Comment (juliensnz) [inline upgrades/schema/Version_2_2_20180321103340_index_products_and_product_models.php]: we should also re-index product and product model no?

Comment (juliensnz) [reply upgrades/schema/Version_2_2_20180321103340_index_products_and_product_models.php]: That's kind of fucked up as the reset will remove indexed proposals no?

Comment (SamirBoulil) [reply upgrades/schema/Version_2_2_20180321103340_index_products_and_product_models.php]: product-model reindex is just above. You are right about the proposals. The migration of proposals has to be executed after this migration. I'll change it

Comment (SamirBoulil) [reply upgrades/schema/Version_2_2_20180321103340_index_products_and_product_models.php]: Fixed. I asked the collaboration team to create the command to reindex proposals as you suggested.

**Pull request merged**